c10/ovrsource_defs.bzl

```diff
@@ -65,7 +68,9 @@ def define_c10_ovrsource(name, is_mobile):
         visibility = [
             "//xplat/caffe2/c10:c10_ovrsource",
         ],
-        deps = select({
+        deps = [
+            "//third-party/rt",
+        ] + select({
```

Comment:
@kit1980 would you be able to help on the meta-internal side? I think this might fix it but not idea if it works.

Reply:
I can try but not today.

Reply:
@kit1980 any updates on this?

Reply:
ping @kit1980

Reply:
I'll look today.